modules/social_features/social_group/modules/social_group_default_route/src/EventSubscriber/RedirectSubscriber.php

```diff
@@ -66,52 +68,53 @@ public static function getSubscribedEvents() {
   public function groupLandingPage(RequestEvent $event) {
 
     // First check if the current route is the group canonical.
-    $routeMatch = $this->currentRoute->getRouteName();
+    $route_name = $this->currentRoute->getRouteName();
 
     // Not group canonical, then we leave.
     if (
-      $routeMatch !== 'entity.group.canonical' &&
-      $routeMatch !== 'social_group_default.group_home'
+      $route_name !== 'entity.group.canonical' &&
```

Comment:
I see a reason to create a constant for something when it using two times and more but this one uses only once so do you still think that constant is needed?

Reply:
It's more like a "practice" since you already added for the above routes, this would standardise the way you are dealing with those strings, but it's just a suggestion.

Reply:
Done.